rel-eng/fedora-pulp.repo

```diff
@@ -63,3 +31,11 @@ baseurl=https://repos.fedorapeople.org/repos/pulp/pulp/testing/2.6/fedora-$relea
 enabled=0
 skip_if_unavailable=1
 gpgcheck=0
+
+# Version 2.7 Testing Builds
+[pulp-2.7-testing]
+name=Pulp 2.7 Testing Builds
+baseurl=https://repos.fedorapeople.org/repos/pulp/pulp/testing/automation/2.7/dev/fedora-$releasever/$basearch/
+enabled=0
+skip_if_unavailable=1
+gpgcheck=0
```

Comment:
@beav Oh I didn't realize they weren't signed. If we want to sign them, OK, but I'm also ok publishing the .repo file with `gpgcheck=0`. @barnabycourt what do you recommend?